pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractReplicator.java

```diff
@@ -47,7 +47,7 @@
     protected volatile ProducerImpl producer;
 
     protected final int producerQueueSize;
-    protected final ProducerBuilder producerBuilder;
+    protected final ProducerBuilder<byte[]> producerBuilder;
```

Comment:
The previous file `.swp` was mistakenly added to the PR.

Reply:
oops.